exponentOperation.h

```diff
@@ -8,7 +8,7 @@ template <typename T>
 class ExponentOperation : public Operation<T> {
 
     public:
-        Tensor<T> *t1; // generally an operation had two operands
+        Tensor<T> *t1,*t3; // generally an operation had two operands
```

Comment:
nit: move t3 to newline for better clarity, and add a comment indicating the input and temp/output tensors